pkgs/tools/package-management/akku/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, stdenv, fetchurl, autoreconfHook, pkg-config, guile, curl, substituteAll }:
+
+stdenv.mkDerivation rec {
+  pname = "akku";
+  version = "1.1.0";
+
+  src = fetchurl {
+    url = "https://gitlab.com/akkuscm/akku/-/archive/v${version}/akku-v${version}.tar.gz";
+    sha256 = "1ddxg2vk7rl9a7mqix05qxpvric5p5rl6w4qcm8abvgg39w2nbjp";
+  };
+
+  patches = [
+    (substituteAll {
+      src = ./hardcode-libcurl.patch;
+      libcurl = "${curl.out}/lib/libcurl${stdenv.hostPlatform.extensions.sharedLibrary}";
+    })
+  ];
+
+  nativeBuildInputs = [ autoreconfHook pkg-config ];
+
+  buildInputs = [ guile ];
```

Comment:
are you sure guile is not in  `nativeBuildInputs`?

Reply:
yes, guile is required at runtime. Akku is a Scheme script.